advisor/build.gradle.kts

```diff
@@ -26,6 +26,7 @@ plugins {
 dependencies {
     api(project(":clients:nexus-iq"))
     api(project(":clients:oss-index"))
+    api(project(":clients:osv"))
```

Comment:
One more thing: Shouldn't this commit be set to auto-close https://github.com/oss-review-toolkit/ort/issues/3599 via the commit message?

Reply:
> One more thing: Shouldn't this commit be set to auto-close #3599 via the commit message?

Scratch that, I overlooked

> A first iteration for https://github.com/oss-review-toolkit/ort/issues/3599 with a couple of things left as TODO.

in the OP.